package.json

```diff
@@ -27,7 +27,7 @@
     "test:coverage": "cross-env NODE_PATH=. NODE_ENV=test istanbul cover _mocha 'test/unit/**/*.test.js' --dir test/unit/coverage/html",
     "test:crossbrowser": "cross-env NODE_PATH=. NODE_ENV=test codeceptjs run-multiple ${BROWSER_GROUP:-'--all'} -c test/e2e/saucelabs.conf.js --steps --reporter mocha-multi --verbose",
     "test:a11y": "cross-env NODE_PATH=. NODE_ENV=a11y LOG_LEVEL=ERROR mocha --exit test/accessibility/a11y.js --exit --timeout 60000 --reporter mochawesome --reporter-options reportFilename=a11y,inlineAssets=true,reportTitle=submit-your-appeal",
-    "sonar-scan": "sonar-scanner -Dsonar.projectName='SSCS :: Submit Your Appeal frontend' -Dsonar.projectKey='SSCSSYAF' -Dsonar.sources=. -Dsonar.exclusions=app.js,app-insights.js,server.js,node_modules/**,mocks/**,policy-pages/**,test/**,assets/**,dist/**,mochawesome-report/**,middleware/draftAppealStoreMiddleware.js -Dsonar.javascript.lcov.reportPaths=test/unit/coverage/html/lcov.info -Dsonar.cpd.exclusions=steps/**",
+    "sonar-scan": "sonar-scanner -Dsonar.projectName='SSCS :: Submit Your Appeal frontend' -Dsonar.projectKey='SSCSSYAF' -Dsonar.sources=. -Dsonar.exclusions=app.js,app-insights.js,server.js,node_modules/**,mocks/**,policy-pages/**,test/**,assets/**,dist/**,mochawesome-report/**,middleware/draftAppealStoreMiddleware.js,components/** -Dsonar.javascript.lcov.reportPaths=test/unit/coverage/html/lcov.info -Dsonar.cpd.exclusions=steps/**",
```

Comment:
why are we ignoring it?

Reply:
because as we said we will be focusing on save and return ?

Reply:
what do you mean? this has nothing to do with the save and return. it's a separate feature and you are ignoring the sonar scan for it. Unless you have a good reason we cannot do this, Izel. Otherwise, if someone from QA team sees it SSCS will be questioned